src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -522,12 +522,14 @@ protected function supportsColors()
                         case '--color=yes':
                         case '--color=force':
                         case '--color=always':
+                        case '--colors=always':
```

Comment:
Does --colors also support yes and force as a value?

Reply:
Nope, it doesn't: https://phpunit.readthedocs.io/en/9.3/textui.html?highlight=--colors#command-line-options

The only three values supported are `auto`, `never` and `always`:
https://github.com/sebastianbergmann/phpunit/blob/19e27be279ab0056984a42f93dace950d9116ba7/src/TextUI/DefaultResultPrinter.php#L65